### PR TITLE
chore: Release python lambda sdk v0.1.12

### PR DESCRIPTION
**Pull request by selcukcihan**
Related to https://linear.app/serverless/issue/SC-708#comment-a402e124

### CODE REVIEW

**Comment by selcukcihan** [comment]
@medikoo ready for review please

File: python/packages/aws-lambda-sdk/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## 0.1.12 (2023-04-05)
+
+### Bug Fixes
+
+- Fix customer module initialization
```
Comment (medikoo) [inline python/packages/aws-lambda-sdk/CHANGELOG.md:9]: Let's also add here `Revert breaking rename of `serverless_sdk` into `sls_sdk` (it belongs to bug fixes as it was a breaking change that was published with a patch bump)

**Review by medikoo — CHANGES_REQUESTED** [review]

File: python/packages/aws-lambda-sdk/CHANGELOG.md (hunk 1 above)
Comment (selcukcihan) [reply python/packages/aws-lambda-sdk/CHANGELOG.md:9]: Done 👍

**Review by medikoo — APPROVED** [review]
Thank you 👍

**Pull request merged**